src/Data/Ord.purs

```diff
@@ -149,3 +151,13 @@ between low hi x
   | x < low = false
   | x > hi = false
   | true = true
+
+-- | The absolute value function. `abs x` is defined as `if x > zero then x
+-- | else negate x`.
+abs :: forall a. (Ord a, Ring a) => a -> a
+abs x = if x > zero then x else negate x
```

Comment:
Should we use `x >= zero` here so we don't end up with signed zero weirdness? It would probably make sense for `abs 0` to return `0` rather than `-0`!